service/models/employer/employer_job.py

```diff
@@ -27,8 +26,18 @@ class EmployerJob(BaseModel):
     job_requirements: Optional[List[str]] = Field(description="Job requirmenets needed")
     job_salary: Optional[int] = Field(description="Optional job salary")
     job_experience_needed: Optional[str] = Field(description="Experience needed for the job")
-    created_time: Optional[Decimal] = Field(description="Creation time of the employer job")
-
+    created_time: Optional[datetime] = Field(description="Creation time of the employer job",
+                                             default_factory=datetime.utcnow)
+
+    class Config:
+        json_encoders = {
+            datetime: lambda v: v.strftime("%FT%T%z")
+        }
+
+    def create_employer_job_item(self) -> Dict:
```

Comment:
this is not a pydentic standard method, could you override the standard? .dict(exclude_none=True)